itou/cities/management/commands/import_cities.py

```diff
@@ -61,15 +99,25 @@ def handle(self, dry_run=False, **options):
         with open(CITIES_JSON_FILE, "r") as raw_json_data:
 
             json_data = json.load(raw_json_data)
-            total_len = len(json_data)
-            last_progress = 0
 
-            for i, item in enumerate(json_data):
+            # Transform list into convenient dictionary.
+            json_data = {(item["nom"], item["code"]): item for item in json_data}
 
-                progress = int((100 * i) / total_len)
-                if progress > last_progress + 5:
-                    self.stdout.write(f"Creating cities… {progress}%")
-                    last_progress = progress
+            for item in MISSING_COORDS:
+                key = (item["nom"], item["code"])
+                if key in json_data:
+                    # The city of some missing coords is present in cities.json.
+                    # e.g. Miquelon-Langlade
+                    json_data[key]["centre"] = item["centre"]
+                    continue
```

Comment:
`continue` c'est plus idiomatique que `if / else` ?

Reply:
J'aime bien parce que ça fait de l'indentation en moins.

```
if condition:
    stuff
    continue
stuff
```

vs

```
if condition:
    stuff
else:
    stuff
```

Reply:
L'intérêt se voit mieux quand il y a plusieurs `if/continue` de suite dans le même `for`, ou bien si le `stuff` post continue est gros et lui-même très indenté.